.gitignore

```diff
@@ -3,3 +3,7 @@
 **/__pycache__/
 db.sqlite3
 *.code-workspace
+.idea/*
+emis/*
+authentication/*
+helpers/*
```

Comment:
Is this ignoring all files in the emis, authentication, and helpers folder?

Reply:
Yes. inorder not to overwrite the settings in the main branch